README.md

```diff
@@ -11,157 +18,79 @@ Client:
 
 Server:
 
-- Node.js,
+- Node,
```

Comment:
Is mongoose the main dependency?

Reply:
I don't understand the question. Please clarify

Reply:
You list main EOC's dependencies. I only wonder if Mongoose is such a dependency. If so we should list it also.

Reply:
Mongoose is MongoDB client.
App is MERN Stack so Mongo Express React Node are the most important